ios/Podfile

```diff
@@ -1,12 +1,38 @@
 # Uncomment the next line to define a global platform for your project
 platform :ios, '9.0'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 target 'RNFirebaseStarter' do
   # Pods for RNFirebaseStarter
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
 
+  # Pods for RN-firebase-starter
+  pod 'React', :path => '../node_modules/react-native/'
+  pod 'React-Core', :path => '../node_modules/react-native/React'
+  pod 'React-DevSupport', :path => '../node_modules/react-native/React'
+  pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
+  pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
+  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
+  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
+  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
+  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
+  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
+  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
+  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
+  pod 'React-RCTWebSocket', :path => '../node_modules/react-native/Libraries/WebSocket'
+  pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
+  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
+  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
+  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
+  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
   # Required by RNFirebase
+  pod 'RNFirebase', path: '../node_modules/react-native-firebase/ios'
```

Comment:
This is actually not required in auto-linking. I no longer have this line, it autolinks in. But the rest of them are important (any `Firebase/xxx` module you need)

Reply:
I did try and remove this, cleared all derived data and build folder. Was not able to build without - screenshot below:

<img width="434" alt="Screen Shot 2019-09-09 at 9 37 39 PM" src="https://user-images.githubusercontent.com/30823997/64578471-83483200-d34d-11e9-8928-e11b6da259a3.png">

Reply:
This is really the only remaining thing I'm hung up on. I definitely don't have this and it definitely works. I wonder if the project is still linked and needs to be unlinked in order for the auto-linking to take effect during pod install? That might explain the RCTLog thing too (which is odd, to me)

Wait! I think I have it! I believe it's missing use_native before the end of the Podfile https://github.com/react-native-community/rn-diff-purge/blob/release/0.60.5/RnDiffApp/ios/Podfile#L35 so it's not linking. I bet if you add that, you can take out the core module (it should auto-link) and it might fix RCTLog too?

Reply:
You are correct. Both issues resolved! Was missing the RNFirebaseStarterTests target in the podfile

Reply:
I also removed the reference to this issue in the readme file as it's no longer needed.